rts/Map/ReadMap.h

```diff
@@ -286,16 +291,18 @@ extern MapDimensions mapDims;
 
 inline float CReadMap::AddHeight(const int idx, const float a) { return SetHeight(idx, a, 1); }
 inline float CReadMap::SetHeight(const int idx, const float h, const int add) {
-	float& x = (*heightMapSyncedPtr)[idx];
+	float& heightRef = (*heightMapSyncedPtr)[idx];
 
 	// add=0 <--> x = x*0 + h =   h
 	// add=1 <--> x = x*1 + h = x+h
-	x = x * add + h;
+	float newHeight = heightRef * add + h;
+	if (newHeight == heightRef)
```

Comment:
```
updateHeightBounds |= (newHeight != heightRef);
return (heightRef = newHeight);
```

or

`return (updateHeightBounds |= (newHeight != heightRef), heightRef = newHeight);`

Reply:
I personally think such style  (`return (x = y);` or `return (x, y);`) hurts readability, but I guess it's matter of preference.
Committed 1st option.